src/cljc/witan/schema.cljc

```diff
@@ -19,3 +20,101 @@
   "validation for /login"
   {(s/required-key :username) (s/both (length-greater 5) (is-an-email))
    (s/required-key :password) (length-greater 5)})
+
+(def IdType
+  s/Str)
```

Comment:
true, didn't think about that.
But it seems the prismatic people did:
https://github.com/Prismatic/schema/blob/master/src/cljx/schema/coerce.cljx#L87